blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -609,3 +626,15 @@ def test_greatest(gl_data):
 def test_least(gl_data):
     u, t = gl_data
     assert odo(least(u.a.max(), t.a.max()), int) == 1
+
+
+def test_coalesce(sqla):
+    t = symbol('t', discover(sqla))
+    assert (
+        odo(compute(coalesce(t.B, -1), {t: sqla}), list) ==
```

Comment:
nit: for consistency, we've been using `compute(expr, sql_data, return_type=list)` for simple cases like this rather than an explicit `odo` call.